### ISSUE TITLE
[IMPORTANT] AUTO REPAIR ITEM WHEN I DROP IT

### ISSUE THREAD

**Issue by LordZargum** [CitizensDev/Citizens#1061]
Hi,
I have a problem whit this plugin. When I drop my item on the floor, server automatically repair item! Can anyone help me??

**Comment by Linx04** [comment]
Are you sure the item is really being damaged? Remember that damaging an npc is cancelling the event server side, client is still thinking the weapon is damaged, so it will show it as broken until updated (throwing and picking up = updating)

**Comment by cdelashmutt** [comment]
This does seem to cause the item to break, however, if you click on an NPC with an item right at the point of breaking.

**Comment by Linx04** [comment]
No, Items are not taking damage (at least not with denizens because event is cancelled). Weapons are not being broken, you see the animation and the item dissapear in the client, but the item is still there in server side, you just need to update your inventory (logging in/out, or sometimes just clicking with your bare hand in the slot where the tool was)
